### PR TITLE
Adds broadcast image

### PR DESCRIPTION
**Pull request by ZmagoD**
https://github.com/roschaefer/rundfunk-mitbestimmen/issues/42

### CODE REVIEW

File: frontend/app/styles/app.css [hunk 1]
```diff
@@ -117,6 +117,17 @@ p {
   visibility: hidden;
 }
 
+.bradcast-image-container {
```
Comment (roschaefer) [inline frontend/app/styles/app.css]: typo: `bradcast` => `broadcast`

File: frontend/app/templates/components/broadcast/broadcast-details.hbs [hunk 2]
```diff
@@ -9,6 +9,11 @@
     <div class="title header detail">
       {{ broadcast.title }}
     </div>
+    {{#if broadcast.urlPresent }}
+      <div class="bradcast-image-container">
```
Comment (roschaefer) [inline frontend/app/templates/components/broadcast/broadcast-details.hbs]: Use `class="image"` here. Both `card` and `image` are css classed defined by Semantic UI, see here: https://semantic-ui.com/views/card.html - then you don't need to write custom css.

File: frontend/app/models/broadcast.js [hunk 3]
```diff
@@ -14,6 +15,10 @@ export default DS.Model.extend({
     return this.get('impressions.firstObject.response');
   }),
 
+  urlPresent: Ember.computed('pictureUrl', function() {
```
Comment (roschaefer) [inline frontend/app/models/broadcast.js]: You don't need that helper method. In handlebars you can do `{{#if broadcast.pictureUrl}}`

File: frontend/translations/en.yaml [hunk 4]
```diff
@@ -35,6 +35,9 @@ broadcast-form:
     label: Description
     placeholder: >
       ..for people who do not know this broadcast. Short and to the point!
+  pictureUrl:
+    label: Image Url
+    placeholder: http://broadcast
```
Comment (roschaefer) [inline frontend/translations/en.yaml]: change to:
`placeholder: http://www.german-mediathek.de/url/of/an/image/`

File: frontend/translations/de.yaml [hunk 5]
```diff
@@ -35,6 +35,9 @@ broadcast-form:
     label: Beschreibung
     placeholder: >
       ..für die Beitragszahler, die die Sendung nicht kennen. Kurz und knackig!
+  pictureUrl:
+    label: Bild URL
+    placeholder: http://beschreibung_bild.com
```
Comment (roschaefer) [inline frontend/translations/de.yaml]: change to:
`placeholder: http://www.eine-mediathek.de/url/von/einem/Bild`

File: backend/db/migrate/20171012180512_add_picture_url_to_broadcasts.rb [hunk 6]
```diff
@@ -0,0 +1,5 @@
+class AddPictureUrlToBroadcasts < ActiveRecord::Migration[5.1]
+  def change
+    add_column :broadcasts, :picture_url, :string
```
Comment (roschaefer) [inline backend/db/migrate/20171012180512_add_picture_url_to_broadcasts.rb]: It's a matter of taste, but is `image` maybe a better name? :thinking:

Comment (roschaefer) [reply backend/db/migrate/20171012180512_add_picture_url_to_broadcasts.rb]: https://english.stackexchange.com/questions/23716/whats-the-difference-between-a-picture-and-an-image

File: features/find-broadcasts/detail_view.feature [hunk 7]
```diff
@@ -7,8 +7,8 @@ Feature: Link to broadcast details page on every broadcast card
   Background:
     Given I am logged in
     And we have these broadcasts in our database:
-      | Title                | Medium | Station   | Created at | Updated at | Description                                                    |
-      | Medienmagazin        | Radio  | radioeins | 2017-07-27 | 2017-08-01 | Welche Zukunft hat die ARD? Antworten gibt's im Medienmagazin. |
+      | Title                | Medium | Station   | Created at | Updated at | Description                                                    | Picture url                                    |
+      | Medienmagazin        | Radio  | radioeins | 2017-07-27 | 2017-08-01 | Welche Zukunft hat die ARD? Antworten gibt's im Medienmagazin. | https://www.zdf.de/assets/teamfoto-102~768x432 |
```
Comment (roschaefer) [inline features/find-broadcasts/detail_view.feature]: we could avoid an unnecessary call of our CI server to [zdf.de](http://zdf.de) if we just use an image url of `localhost`

**Comment by ZmagoD** [comment]
I will make a new clean PR on this one :smile:

**Comment by roschaefer** [comment]
I can help you out on rebasing if you want.

**Pull request closed without merging**